package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "lint": "standard",
     "test": "npm-run-all -p -r mock test:ava",
-    "test:ava": "ava --verbose",
+    "test:ava": "ava --verbose --serial",
```

Comment:
musing: perhaps we could give each test it's own store location to get the isolation back and allow parallel tests

Reply:
Yes. I spent _so_ long tracking this issue down.